types/index.d.ts

```diff
@@ -271,6 +271,9 @@ interface WebLNProvider {
   sendPayment(paymentRequest: string): Promise<SendPaymentResponse>;
   signMessage(message: string): Promise<SignMessageResponse>;
   verifyMessage(signature: string, message: string): Promise<void>;
+  on(eventName: string, listener: () => void): this;
+  off(eventName: string, listener: () => void): this;
+  emit(eventName: string, ...args: any[]): boolean;
```

Comment:
emit should not be called externally, can you remove this one?

Reply:
resolved

Reply:
@pavanjoshi914 thanks.

Sorry, I just realized, on and off should also be optional.

Reply:
nice catch! thanks. made those changes